README.md

```diff
@@ -25,42 +25,36 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
 
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[your\_project\_name\]  Budget App](#-your_project_name--budget-app)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [your_project_name] <a name="about-project"> Budget App</a>
 
```

Comment:
**NOTE:** The optional points are only optional because they were not mentioned by the previous reviewer, but I highly recommend that you take them into consideration.

There are a few things you can improve about your README to make it more professional.

## [REQUIRED]

As requested by the previous reviewer. Kindly fill in all the dummy sections with the relevant content.

- You do not have a project description, kindly write a well-detailed description of your project.

![image](https://user-images.githubusercontent.com/38088818/215269344-ce39c9a8-2488-4082-a68b-c147d539e2b5.png)

- The `Show your support` section is not documented. Kindly write the relevant information for this section.

![image](https://user-images.githubusercontent.com/38088818/215269420-b75df450-36fe-4d5f-960c-81f14790bf78.png)

- Since the FAQ section is OPTIONAL and you do not have any information for it. Kindly remove it rather than leave it empty.

![image](https://user-images.githubusercontent.com/38088818/215269502-2d2acb62-8dad-4e03-b449-ea1ebb4b517e.png)

- Please make sure that you do not have any broken links on your README. The link to your license is broken, kindly fix it and make sure that users are redirected to the relevant license file.

![image](https://user-images.githubusercontent.com/38088818/215269620-db659080-3d05-4f03-b565-ee474d1432ee.png)

## [OPTIONAL] 

- Please improve your project title on both the table of contents and the main section by removing the example text in square brackets '[your_project_name]'.

![image](https://user-images.githubusercontent.com/38088818/215268853-26c9a6c0-866b-40c4-95da-41f42c8ad842.png)

- Kindly remove all the description texts under each section that came with the template, they are meant to guide you on how to modify each section. They should not be in the final documentation of your project.

For example:
![image](https://user-images.githubusercontent.com/38088818/215269014-2d09798e-f3c7-4e5a-bb15-6aafc20ea62e.png)
![image](https://user-images.githubusercontent.com/38088818/215269047-644930b4-88ad-4863-9490-183a3aa48d34.png)
![image](https://user-images.githubusercontent.com/38088818/215269075-92a07196-0c26-4b46-91ac-bacaf8edf3f6.png)

And so forth... Be sure to remove the rest that are not included on the screenshots above.